### PR TITLE
Move KerasTuner, KerasCV, KerasNLP higher

### PR DESCRIPTION
**Pull request by sachinprasadhs**
Moved KerasTuner, KerasCV, KerasNLP to top for more visibility.
Closes: https://github.com/keras-team/keras-io/issues/857
image

### CODE REVIEW

File: scripts/master.py [hunk 1]
```diff
@@ -529,18 +529,6 @@
...
-        },
-        {
-            "path": "governance",
-            "title": "Community & governance",
-        },
-        {
-            "path": "contributing",
-            "title": "Contributing to Keras",
-        },
         {
```
Comment (mattdangerw) [inline scripts/master.py:532]: I might find things more scannable if we grouped things better. And potentially added a caption for all our libraries (similar to what we have for KerasCore?).

About
Getting Started
Guides
Examples
API Reference
Keras Core: Keras for TensorFlow, JAX, and PyTorch
Keras Tuner: Hyperparameter Tuning
Keras CV: Computer Vision Workflows
Keras NLP: Natural Language Workflows

Then we are keeping "like next to like." E.g. guides next to examples. Ecosystem libraries next to each other. @fchollet what do you think?

Comment (mattdangerw) [reply scripts/master.py:532]: Oh also, let's rename the `Keras API Reference` section to just `API Reference`. It's implied that everything is Keras, and we also list the domain package APIs there.

Comment (sachinprasadhs) [reply scripts/master.py:532]: Thanks for reviewing, I made the changes as you suggested, please review the below screenshot before I make commit.

image

**Review by fchollet — APPROVED** [review]
This looks like a good idea. Thanks for the PR.

**Pull request merged**